src/core/inlines.js

```diff
@@ -147,7 +147,7 @@ export function run(conf) {
       "\\b(?:NOT\\s+)?REQUIRED\\b",
       "\\b(?:NOT\\s+)?RECOMMENDED\\b",
       "\\bOPTIONAL\\b",
-      "(?:{{2}\\s*.*\\s*}{2})", // inline IDL references,
+      "(?:{{[^}}]+}})", // inline IDL references,
```

Comment:
As @sidvishnoi said this should really be `[^}]`, because exclude `}` twice is redundant.

Reply:
oh whoops. Merged too quick.

Reply:
I'll send fix with https://github.com/w3c/respec/issues/2259